dev/com.ibm.ws.jca_fat_duplicates/fat/src/com/ibm/ws/jca/fat/duplicates/FATSuite.java

```diff
@@ -10,22 +10,15 @@
  *******************************************************************************/
 package com.ibm.ws.jca.fat.duplicates;
 
-import org.junit.ClassRule;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 import org.junit.runners.Suite.SuiteClasses;
 
-import componenttest.rules.repeater.EmptyAction;
-import componenttest.rules.repeater.FeatureReplacementAction;
-import componenttest.rules.repeater.RepeatTests;
+import componenttest.custom.junit.runner.AlwaysPassesTest;
 
 @RunWith(Suite.class)
-@SuiteClasses(DuplicateResourceAdaptersTest.class)
+@SuiteClasses({ AlwaysPassesTest.class,
+                DuplicateResourceAdaptersTest.class })
 public class FATSuite {
-    /*
-     * EE7 and EE9 will be run with lite and full fat since this fat is very quick.
-     */
-    @ClassRule
-    public static RepeatTests r = RepeatTests.with(new EmptyAction())
-                    .andWith(FeatureReplacementAction.EE9_FEATURES());
+    //this tests the EE10 connectors version 2.1 resource adapter and should not be repeated for previous versions
```

Comment:
Are we losing test coverage for previous JCA / Connectors versions?

Reply:
No, we discussed this in scrum and Nathan decided we should convert this project because the function for this test, besides now ensuring a 2.1 version resource adapter deploys, it to ensure that you can't deploy 2 identical resource adapters and that function doesn't change between versions.